build.gradle

```diff
@@ -221,6 +221,7 @@ dependencies {
     testCompile "org.testfx:testfx-core:4.0.17-alpha-SNAPSHOT"
     testCompile "org.testfx:testfx-junit5:4.0.17-alpha-SNAPSHOT"
     testCompile "org.hamcrest:hamcrest-library:2.2"
+    testCompile group: 'org.assertj', name: 'assertj-core', version: '3.14.0'
```

Comment:
I think we've decided at some point to only use vanilla plain assert statements. (I think the reason was to reduce the burden for new comers to learn different styles of assertions.)

Reply:
I remember somehow. Now, I would follow the argumentation by then (which is also stated by Google): "A better way to structure these assertions is to use a fluent API"

Nevertheless, I would switch to Google Truth - https://opensource.googleblog.com/2019/07/truth-10-fluent-assertions-for-java-and.html

As next step, I'll try to convert it to basic Java code. If that does not work, I will use Google Truth and explain why.